packages/flutter_driver/lib/src/error.dart

```diff
@@ -65,26 +93,33 @@ class LogRecord {
 
 /// Package-private; users should use other public logging libraries.
 class Logger {
+  /// Creates a new logger.
   Logger(this.name);
 
+  /// Logger name.
```

Comment:
Needs more detail. Any time you feel tempted to write docs that could have been written by someone who knew nothing other than the fully-qualified name of the identifier, you need more detail. :-)